bands/templates/bands/agreement.html

```diff
@@ -55,13 +55,14 @@
 
 <h3>Your Information:</h3>
 <form method="post" action="agreement" class="form-horizontal" role="form">
-    <input type="hidden" name="id" value="{{ band.id }}" />
+    <input type="hidden" name="band_id" value="{{ band.id }}" />
+    <input type="hidden" name="id" value="{{ band.info.id|default:None }}" />
```

Comment:
Since we're passing ``band_info`` to this template, we can just say ``value="{{ band_info.db_id }}`` which is shorthand for "the id if this has been saved to the database, otherwise None".  This is defined at
https://github.com/magfest/ubersystem/blob/master/uber/models.py#L270
and is commonly used in our forms.